scripts/examples/typescript/README.md

```diff
@@ -0,0 +1,41 @@
+# A Typescript bindings for XenAPI
```

Comment:
typo here and below
```suggestion
# Typescript bindings for XenAPI
```

Reply:
I would avoid the use of "bindings" completely, first because we don't provide real bindings and secomnd because this is not the terminology used for the rest od the SDK. We can use "module" until we provide a full blown SDK.